### PR TITLE
Add images to Users and Products

### PR DESCRIPTION
**Pull request by maximihajlov**
This PR adds a new field `image` to User, so now users can provide their avatar while registration. 

It also adds `images` to Product, so now sellers can add and remove product images.

### CODE REVIEW

**Review by adrian-lui — APPROVED** [review]
Approved, thanks for the great work! I will change frontend struct accordingly for images.

File: backend/services/products/src/main/java/pw/mer/letsplay/web/ProductController.java [hunk 1]
```diff
@@ -36,82 +39,97 @@ public List<Product> index() {
...
         @Size(min = 3, max = 1000, message = "Description must be between 3 and 1000 characters long")
         private String description;
 
+        @JsonProperty("price")
         @NotNull(message = "Price is mandatory")
         @PositiveOrZero(message = "Price can not be negative")
         private Double price;
+
+        @JsonProperty("images")
+        private List<String> images = List.of();
```
[the remark this answers is not in the document]
Comment (maximihajlov) [reply backend/services/products/src/main/java/pw/mer/letsplay/web/ProductController.java:66]: Array of strings makes more sense to me. It's always array of strings, even if there's only one image.

File: backend/services/products/src/main/java/pw/mer/letsplay/web/ProductController.java [hunk 2]
```diff
@@ -36,82 +39,97 @@ public List<Product> index() {
...
 
     @GetMapping("/{id}")
+    @SecurityRequirement(name = "Authentication", scopes = {"products:admin", "products:write"})
     public Product getById(@PathVariable String id) {
         return productRepo.findById(id).orElseThrow(() ->
                 new ResponseStatusException(NOT_FOUND, NOT_FOUND_MESSAGE));
     }
 
     @Setter
     public static class AddProductRequest {
```
[the remark this answers is not in the document]
Comment (maximihajlov) [reply backend/services/products/src/main/java/pw/mer/letsplay/web/ProductController.java]: Added in fc1278d

**Comment by maximihajlov** [comment]
Thanks for the review!

**Pull request merged**